airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/xmin/XminCtidUtils.java

```diff
@@ -84,14 +85,15 @@ public static StreamsCategorised<XminStreams> categoriseStreams(final StateManag
         .map(Jsons::clone)
```

Comment:
Can we modify CtidUtils.identifyNewlyAddedStreams to filter for incremental streams?

Reply:
done